### PR TITLE
1658 messages header

### PR DESCRIPTION
**Pull request by matthopson**
Adds blue header to dashboard messages

Screen Shot 2019-05-14 at 2 35 01 PM

### CODE REVIEW

File: web-client/src/views/HeaderDashboardInternal.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ({ chooseWorkQueueSequence, unreadCount, workQueueHelper }) => {
+    const currentQueueTitle = `${
+      workQueueHelper.showIndividualWorkQueue ? 'My' : 'Section'
+    } Messages`;
+    const currentBoxView = workQueueHelper.showInbox ? 'inbox' : 'outbox';
+    return (
+      <div className="big-blue-header">
+        <div className="grid-container">
+          <h1 tabIndex="-1">{currentQueueTitle}</h1>
+          <span className="unread" aria-label="undread work item count">
```
Comment (kkoskelin) [inline web-client/src/views/HeaderDashboardInternal.jsx:21]: `undread` opps

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/HeaderDashboardInternal.jsx (hunk 1 above)
Comment (matthopson) [reply web-client/src/views/HeaderDashboardInternal.jsx:21]: ?

Comment (kkoskelin) [reply web-client/src/views/HeaderDashboardInternal.jsx:21]: I believe it is spelled "unread".
Anyhow I know it wasn't your original typo.  I thought I'd fixed it another time, perhaps it got lost in a merge.

Comment (matthopson) [reply web-client/src/views/HeaderDashboardInternal.jsx:21]: Ohhh! haha. I see now. I'll get it fixed up. Good catch.

**Pull request merged**